server/controller.js

```diff
@@ -317,8 +317,8 @@ controller.updateIncidentImage = async (req, res, next) => {
 controller.updateIncidentDetails = async (req, res, next) => {
   console.log("req.body", req.body);
```

Comment:
I think once the functions are working as intended, we can remove the console logs :)